src/Sql/Sql.Test/ScenarioTests/ManagedDatabaseCrudScenarioTests.cs

```diff
@@ -63,5 +63,19 @@ public void TestRestoreManagedDatabase()
         {
             RunPowerShellTest("Test-RestoreManagedDatabase");
         }
+
+        [Fact(Skip = "Setup time is long (more than 10 hours)")]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
```

Comment:
I guess the skip reason only makes sense for LiveOnly category tests. Once recorded - the test time should't take long.

Reply:
In case other person will do record test